src/main/java/GUI.java

```diff
@@ -1,14 +1,49 @@
+package GUI;
+
 import javax.swing.*;
+import javax.swing.border.Border;
```

Comment:
line 3 already imports everything from swing, so we probably don't need lines 4 and 5

Reply:
Not sure, but Inteliij warns me to do so.